src/components/pages/Offer.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useLayoutEffect } from 'react'
+
+import { Container, Col, Row } from 'react-bootstrap'
+
+import OfferContact from './../organisms/offerContact'
+
+import logo from './../../images/logo-placeholder.svg'
+
+import './pages.scss'
+
+const Offer = () => {
+  const [offer, setOffer] = useState({})
+
+  useLayoutEffect(() => {
```

Comment:
Jakiś konkretny powód, dlaczego używasz tutaj `useLayoutEffect`? Czy to jest spowodowane renderowanie mapy?